beacon-chain/powchain/check_transition_config.go

```diff
@@ -116,7 +116,7 @@ func (s *Service) handleExchangeConfigurationError(err error) {
 		log.WithError(err).Error(configMismatchLog)
 		return
 	}
-	log.WithError(err).Error("Could not check configuration values between execution and consensus client")
+	log.Errorf("Could not check configuration values between execution and consensus client: %v", err)
```

Comment:
Please avoid using log templates with errors in them. It makes it harder to collect and monitor errors in some infrastructure. 

Instead of changing the log, could you investigate where the stack trace is coming from in the error change?